utils/adapters/firebase.ts

```diff
@@ -34,25 +31,11 @@ export const FirebaseAdapter = (firebase: FirebaseApp): Adapter => {
       return tll;
     },
 
-    createProfile: async (pubkey: string) => {
-      const docRef = db.doc(`profiles/${pubkey}`);
-      const nonce = newNonce().toString();
-      await docRef.set({
-        pubkey: pubkey, // redundant
-        nonce,
-        ttl: +new Date() + 300000, // now + 5min
-      });
-      return nonce;
-    },
+    saveSigninAttempt: async (attempt) => {
+      const docRef = db.doc(`profiles/${attempt.pubkey}`);
```

Comment:
I'm thinking we should probably rename the path to signinattempts/{pubkey}

Profile may be a bit misleading